MediaBrowser.WebDashboard/Api/PackageCreator.cs

```diff
@@ -47,16 +48,25 @@ public static bool IsCoreHtml(string path)
             return string.Equals(Path.GetExtension(path), ".html", StringComparison.OrdinalIgnoreCase);
         }
 
-        // Modifies the HTML by adding common meta tags, css and js.
-        public async Task<Stream> ModifyHtml(
-            string path,
+        /// <summary>
+        /// Modifies the source HTML stream by adding common meta tags, css and js.
+        /// </summary>
+        /// <param name="isMainIndexPage">True if the stream contains content for the main index page.</param>
+        /// <param name="sourceStream">The stream whose content should be modified.</param>
+        /// <param name="mode">The client mode ('cordova', 'android', etc).</param>
+        /// <param name="appVersion">The application version.</param>
+        /// <param name="localizationCulture">The localization culture.</param>
+        /// <returns>
+        /// A task that represents the async operation to read and modify the input stream.
+        /// The task result contains a stream containing the modified HTML content.
+        /// </returns>
+        public static async Task<Stream> ModifyHtml(
+            bool isMainIndexPage,
             Stream sourceStream,
             string mode,
             string appVersion,
             string localizationCulture)
```

Comment:
Made this a static method and updated the method signature so we don't have to pass a magic string "dummy.html" here: https://github.com/jellyfin/jellyfin/pull/2601/files#diff-10c7fab5871b1479d1a685f17fa30ccbR234